DoxygenHelp.md

```diff
@@ -18,25 +18,31 @@ Alternatively install doxygen and dot yourself:
 
 ### Top-level CMakelists.txt
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will automatically find the doxygen and dot executables. The up-to-date version of the file can be found here:
+
 https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt
 
+In the CMake GUI, the following options can be selected:
+- DOXYGEN_OPTIONAL_COMMENTED_ONLY: If this is not selected, doxygen will assume that all entities are documented, even if there is no documentation available. 
+- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal" (comments intended for developers only).
+- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. 
+
 ### ConfigureDoxygen.cmake file
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in CMake, CMake will include ConfigureDoxygen.cmake. This file includes all settings for the generation of doxygen documentation for the Open Infra Platform project. Changes to documentation settings should be made here. doxygen_add_docs() creates the GenerateDocumentation target. The up-to-date version of the file can be found here:
 
 https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake
 
-Tags that are more likely to be required to change:
-- DOXYGEN_EXCLUDE: if you want to exclude certain source code directories or files from the documentation, you should add them here.
-- DOXYGEN_INTERNAL_DOCS: set to yes if you want to include comments included for developers in the documentation. 
+A tag that is more likely to be required to change:
+- DOXYGEN_EXCLUDE_PATTERNS: if you want to exclude certain source code directories or files from the documentation, you should add them here.
 
 More settings to be added can be found here:
 http://www.doxygen.nl/manual/index.html
 
 Important: when adding settingsthem in CMake, always use DOXYGEN_ before the standard doxygen tag, e.g. the doxygen tag "PROJECT_BRIEF" has to be "DOXYGEN_PROJECT_BRIEF" in the ConfigureDoxygen.cmake file.
 
 ## Building the documentation
-In Visual studio, build "GenerateDocumentation".
+In Visual studio, build "OpenInfraPlatform.GenerateDocumentation". This target can be found in Visual Studio under OpenInfraPlatform/Commands.
 In your build directory under Doxymentation/html you will then find something called “index.html”. This is a link to the index page of the HTML documentation browser. 
+This index page of the documentation will be opened automatically if the build succeeded.

```

Comment:
This should also be an option to turn off in CMake.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/ed9c681c25ee0e0b1b7b96ba93eb1de6e07f7af3